deploy/test/helm-props/k8s/helm-app.yaml

```diff
@@ -0,0 +1,29 @@
+# This app is created to run end-to-end with the Secrets Provider Job
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: test-env{{ ID }}
+  name: test-env{{ ID }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: test-env{{ ID }}
+  template:
+    metadata:
+      labels:
+        app: test-env{{ ID }}
+    spec:
+      serviceAccountName:  {{ SERVICE_ACCOUNT }}
+      containers:
+      - image: centos:7
```

Comment:
Is this imitate real life scenario?
What is the purpose of having infinity sleep ?

Reply:
> Real life scenario

Which part of it? The OS or sleep infinity?

I want to mock an app up and am less concerned with which types of actions it is performing or what it is used for

Reply:
iirc we had the container run for infinity so it will stay up and we can grep its env to verify that it has the secret from conjur